types/electron-packager/index.d.ts

```diff
@@ -57,9 +58,19 @@ declare namespace electronPackager {
     }
 
     interface ElectronOsXSignOptions {
-        identity?: string;
         entitlements?: string;
         "entitlements-inherit"?: string;
+        "gatekeeper-assess"?: boolean;
+        // see https://github.com/electron-userland/electron-packager/blob/92d09bba34599283a794fd6f24b88470f0cb1074/src/mac.js#L359
+        identity?: string | true;
+        "identity-validation"?: boolean;
+        ignore?: string;
+        keychain?: string;
+        "pre-auto-entitlements"?: boolean;
+        "pre-embed-provisioning-profile"?: boolean;
+        "provisioning-profile"?: string;
+        requirements?: string;
+        type?: string;
```

Comment:
Is there a way to say something like

```typescript
import { SignOptions } from 'electron-osx-sign';

// ...
interface ElectronOsxSignOptions extends SignOptions {
  identity?: string | true;
}
```

and have `identity` get overridden properly by TypeScript? Personally I'd rather have as little copypasta as possible.

Reply:
Oh, the change I described is in #36441? I'm a little confused.

Reply:
@malept #36441 will be merged after this PR.

I'm also against copypasta, but to import `electron-osx-sign`, https://github.com/microsoft/types-publisher/pull/627 needs to be merged. And this sometimes takes a few days... :slightly_smiling_face:

Reply:
@malept is that valid typescript? See this question in [stackoverflow](https://stackoverflow.com/questions/41285211/overriding-interface-property-type-defined-in-typescript-d-ts-file)

What about once the types get published:

```typescript
import { SignOptions } from 'electron-osx-sign';

interface ElectronOsxSignOptions extends Omit<SignOptions, 'identity'>  {
  identity?: string | true;
}
```

Reply:
> is that valid typescript?

I did say "something like" as my TypeScript skills are not very good. @ffflorian answered my question.